test-cypress.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
```

Comment:
@gouyang I dont see this file `test-cypress.sh` used anywhere?

Reply:
it will be used by downstream CI which has the kubevirt-plugin deployed, `test-prow-e2e` is not suitable for downstream as it deploys the plugin.

it also can be used for triggering tests locally.